src/Resources/config/services/form.xml

```diff
@@ -22,6 +22,7 @@
         <service id="bitbag.sylius_blacklist_plugin.form.type.blacklisting_rule" class="BitBag\SyliusBlacklistPlugin\Form\Type\BlacklistingRuleType">
             <argument>%bitbag_sylius_blacklist_plugin.model.blacklisting_rule.class%</argument>
             <argument>%bitbag_sylius_blacklist_plugin.blacklisting_rule_attribute_choices%</argument>
+            <argument type="string">%sylius.model.customer_group.class%</argument>
```

Comment:
Where do we have the parameter? I don't see any placed in PR. Does it exist?

Reply:
This parameter comes from Sylius

Reply:
Okay, good